testing/vcs/test_plot_unstructured_via_boxfill.py

```diff
@@ -0,0 +1,51 @@
+import vcs
+import os,sys
+import cdms2
+
+pth = os.path.join(os.path.dirname(__file__),"..")
+sys.path.append(pth)
+import checkimage
+
+f=cdms2.open(os.path.join(vcs.prefix,"sample_data","sampleCurveGrid4.nc"))
+
+s=f("sample")
+x=vcs.init()
+x.drawlogooff()
+x.setbgoutputdimensions(1200,1091,units="pixels")
+
+import vcs
```

Comment:
These imports (and the checkimage/open below) don't need to be repeated. Accidental copy/paste?